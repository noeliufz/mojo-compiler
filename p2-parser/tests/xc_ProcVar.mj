{
  var p: proc(x:int): int;
  proc P(x:int):int;
  p = P;
}
