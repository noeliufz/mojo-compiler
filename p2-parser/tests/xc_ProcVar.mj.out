{
  var p: proc (
      val x: int;
    ): int;
  proc P (
      val x: int;
    ): int;
  p = P;
}
